File: server/src/app/services/db/userService.ts
// app/services/db/userService.ts
import { prisma } from '@server/db/prisma';

export async function createUser(name: string) {
    return await prisma.user.create({
        data: { name },
    });
}

export async function getUserById(id: number) {
    return await prisma.user.findUnique({
        where: { id },
    });
}

// Add more user-related database functions as needed

File: server-fast/app/db/userService.py
from ..db.models import User
from ..db.database import database

async def create_user(name: str):
    query = User.__table__.insert().values(name=name)
    await database.execute(query)
    return {"name": name}

async def get_user_by_id(id: int):
    query = User.__table__.select().where(User.id == id)
    user = await database.fetch_one(query)
    return user
